packager/react-packager/src/node-haste/DependencyGraph/ResolutionRequest.js

```diff
@@ -385,7 +381,17 @@ class ResolutionRequest {
             );
           });
 
-          return p;
+          return p.catch(error => {
+            if (error.type !== 'UnableToResolveError') {
+              throw error;
+            }
+            throw new UnableToResolveError(
+              fromModule,
+              toModuleName,
+              `Module does not exist in the module map or as these directories:\n  ` +
```

Comment:
The `searchQueue` contains the full path, not just the parent directories.
Should I do `searchQueue.map` with `path.dirname` and then use "in these"?